components/homepage/modal-overlay/use-modal.tsx

```diff
@@ -42,5 +42,5 @@ export default function useModal(modalKey: string) {
     if (!validKeyInLocalStorage(modalKey)) setShouldDisplay(true);
   });
 
-  return [shouldDisplay, closeModal];
+  return [shouldDisplay, closeModal] as const;
```

Comment:
What does `as const` do?

Reply:
If we remove "as const", Typescript infers its type as Array<boolean | () => void>

By using "as const", Typescript understands that this hook is returning a tuple of length 2 with the first element of boolean type and the second element of a function type.